coredb-operator/src/extensions.rs

```diff
@@ -367,6 +390,32 @@ fn extension_plan(have_changed: &[Extension], actual: &[Extension]) -> (Vec<Exte
     (changed, to_install)
 }
 
+/// reconcile extensions between the spec and the database
```

Comment:
moved this to the bottom of the file so its easier to find. its the "top level" command in the file